mgr/src/cmds/pools.cr

```diff
@@ -13,26 +13,32 @@ handler "pool.create" do |args|
   name = args.pos_args[0]
   api_call(args, "Failed to create the Pool") do |client|
     pool = client.create_pool(name)
+
+    handle_json_output(pool, args)
+
     puts "Pool #{name} created successfully"
     puts "ID: #{pool.id}"
   end
 end
 
-command "pool.list", "Kadalu Storage Pools List" do |parser, _|
+command "pool.list", "Kadalu Storage Pools List" do |parser|
```

Comment:
the `command` helper expects a block of two params. Is this working without the second arg?

Reply:
Interestingly this is working fine even without the second parameter to the block. I added a question in the Crystal forum. I will update here once I get the clarification. https://forum.crystal-lang.org/t/how-to-validate-the-block-signature/4370

For now you can change this to

```crystal
command "pool.list", "Kadalu Storage Pools List" do |parser, _args|
```
Starting with `_` for unused args.

Reply:
I think this was recommended by one of the linters.